pkg/cmd/install.go

```diff
@@ -65,6 +67,18 @@ func MakeInstall() *cobra.Command {
 		k3sVersion, _ := command.Flags().GetString("k3s-version")
 		k3sExtraArgs, _ := command.Flags().GetString("k3s-extra-args")
 
+		useFlannelIPsec, _ := command.Flags().GetBool("ipsec")
+		if useFlannelIPsec {
+			re := regexp.MustCompile(`(?m)(--flannel-backend).\w+`)
```

Comment:
I'm finding this confusing to read, what lead to this approach vs just appending the value if required as per the existing code?

Reply:
Thinking on:
- What if the cli user is copy pasting a long convoluted k3s' extra args and in the middle of them there's the `--flannel-backend vxlan` in the middle of it and you also want  the `--ipsec`  option to be on? this enforces the option to be either replaced with the regex in the `k3sExtraArgs` or appended.

Reply:
Can you try to mimic the existing usage of flags please instead? It will be easier to manage.

Reply:
Ok, done it the KISS way 😉